libopenage/error/handlers.cpp

```diff
@@ -21,6 +21,7 @@
 #include <unistd.h>
 #endif
 
+#include "../error/handlers.h"
```

Comment:
[The associated header file comes first](https://github.com/SFTtech/openage/blob/11a54bf781c08ff45e93c8c672c8c08d4969b8de/doc/code_style/mom.cpp#L17).
Also, `#include "handlers.h"`.